source/playbooks/running-playbooks.rst

```diff
@@ -1,41 +1,30 @@
-Running Playbooks
-=================
+Run a playbook
+==============
 
-Use slash commands to initiate playbook runs, make announcements, and get information about a playbook run.
+You can start a playbook run from any channel that you're a member of, using the **Toggle Playbook List** icon. Select the **Run** icon next to a playbook name.
```

Comment:
The phrase "from any channel that you're a member of" makes me thing I can start a run /in/ that channel, but we currently force the user to create a new channel. Perhaps we should default the user to navigating to Playbooks via the global header, finding a playbook, and then running from there?

(Doing it from the RHS is convenient, but maybe shouldn't be the default path?)